README.md

```diff
@@ -110,3 +110,11 @@ It's the main component of the application. It contains the ToolBar and the netw
 ## Deploy to Heroku
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/nexmo-se/video-api-multiparty-toolkit-sample-app.git)
+
```

Comment:
I prefer to not add specific deployment instructions on the README file of this repo as its general purpose. Could you remove it, please?

Reply:
ok no problem